README.md

```diff
@@ -106,6 +112,20 @@ When opening a PR:
 
 Once the PR is opened, the same automated checks from Step 4 will then run on your PR, and your PR will be reviewed in due course. Once these checks pass the PR will be merged.
 
+<a name="promote-chain"></a>
+## Promote a chain to standard
+This process is only possible for chains already in the registry.
+
+Run this command:
+```
+sh scripts/promote-to-standard.sh <chain-id>
+```
+
+This command will:
+* activate the full suite of validation checks for standard chains, including checks on the `ProxyAdminOwner`
+* declare the chain as a standard chain
+* set the `superchain_time`, so that the chain receives future hardforks with the rest of the superchain (baked into downstream OPStack software).
```

Comment:
What if the PR updating this time doesn't get merged before the next OPStack release is cut?

Reply:
In this case the chain in question will not make it into the network flags options for the OPStack software, and would need to wait for the next release.